api/ca.proto

```diff
@@ -55,3 +60,9 @@ message GetRootCACertificateRequest {}
 message GetRootCACertificateResponse {
 	bytes certificate = 1;
 }
+
+message GetUnlockKeyRequest {}
+
+message GetUnlockKeyResponse {
+	bytes unlock_key = 1;
```

Comment:
The algorithm information is in the `MaybeEncryptedRecord` object - I'm not sure it would affect the key (which is just some random data).  If we needed a bigger key, for instance, we should be able to check the length of the byte slice.